ts/Extensions/Drilldown.ts

```diff
@@ -1368,6 +1407,7 @@ if (PieSeries) {
                 (this.chart.drilldownLevels as any)[
                     (this.chart.drilldownLevels as any).length - 1
                 ],
+                dataLabelsGroup = this.dataLabelsGroup,
```

Comment:
Unused variable.

Nothing did complain about the existence of this line. Why? 🤔

Reply:
Oh, okay. Thanks for the info!